flask-bot/unit_tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+import sys
+import os
+
+sys.path.append(os.path.realpath(os.path.dirname(__file__) + "/.."))
```

Comment:
I'm a fan of pathlib for operations like this, nothing that would block this going in but maybe in future

Reply:
Taking your advice and updating this now, next change will resolve this!
You're right, it looks a lot cleaner :P